ncm-nfs/src/main/pan/components/nfs/schema.pan

```diff
@@ -10,7 +10,7 @@ type structure_nfs_exports = {
 type structure_nfs_mounts = {
     'device' : string
     'mountpoint' : string
-    'fstype' : string with match(SELF, '^(nfs4?|panfs|ceph|none)$')
+    'fstype' : choice('nfs', 'nfs4', 'panfs', 'ceph', 'none', 'lustre')
```

Comment:
Can we put the `choice` options in a logical order? I'm OK with the most common coming first, but special types like `none` should be at the end IMHO.

`choice('nfs', 'nfs4', 'ceph', 'lustre', 'panfs', 'none')`

or just go full alphabetic.

Reply:
:+1: for full alphabetic